src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -19120,6 +19152,19 @@ public static void Main()
                 );
         }
 
+        [Fact]
+        public void RecordLoadedInVisualBasicDisplaysAsRecord()
+        {
+            var src = @"
+public record A;
+";
+            var compRef = CreateCompilation(src).EmitToImageReference();
+            var vbComp = CreateVisualBasicCompilation("", referencedAssemblies: new[] { compRef });
+            var symbol = vbComp.GlobalNamespace.GetTypeMember("A");
+            Assert.Equal("record A",
```

Comment:
>record A [](start = 26, length = 8)

Shouldn't a symbol loaded in VB be displayed with VB syntax (`Class A`)?

Reply:
We're forcing to use the C# display visitor, which I didn't realize was a valid scenario.